openpower/configs/hostboot/habanero.config

```diff
@@ -38,3 +38,6 @@ set DISABLE_HOSTBOOT_RUNTIME
 unset SET_NOMINAL_PSTATE
 set HTMGT
 set START_OCC_DURING_BOOT
+
+#PNOR flags
+set PNOR_TWO_SIDE_SUPPORT
```

Comment:
Hostboot currently has as 'TWO_SIDE_SUPPORT'.  Are we ok with putting in as "PNOR_TWO_SIDE_SUPPORT" for now, knowing that it won't do anything until Hostboot is updated?